bin/host/src/bin/host.rs

```diff
@@ -23,14 +23,9 @@ primary thread.
 #[derive(Parser, Serialize, Clone, Debug)]
 #[command(about = ABOUT, version, styles = cli_styles())]
 pub struct HostCli {
-    /// Verbosity level (0-5)
-    /// If set to 0, no logs are printed.
-    /// By default, the verbosity level is set to 3 (info level).
-    #[arg(long, short, default_value = "3", action = ArgAction::Count)]
-    pub v: u8,
-    /// The format of the logs. One of: full, json, pretty, compact.
-    #[arg(long = "logs.format", short = 'f', default_value = "full")]
-    pub logs_format: LogFormat,
+    /// Logging arguments.
+    #[command(flatten)]
+    pub v: LogArgs,
```

Comment:
slight nit - since `v` is no longer just the verbosity command (and instead the flattened `LogArgs`,) we can change this variable name to something more descriptive.